.gitignore

```diff
@@ -0,0 +1 @@
+terraform-provider-cdap
```

Comment:
What does this mean?

Reply:
This just tells the local clients to ignore this file. It's convenient for developers to not have to think about excluding it when you create a PR.

Reply:
Got it, typically you run `go build -o some/path` and export to another directory where the tf config lives, but this is handy if it was built in the current dir.

Maybe add a comment saying `# Locally built provider binary` or something?

Reply:
done.